relayer/core/relay.go

```diff
@@ -122,21 +107,26 @@ func NewRelay() (*Relay, error) {
 			subMessages = make(chan []chain.Message, 1)
 		}
 
-		err = ethChain.SetReceiver(subMessages, nil, dbMessages, beefyMessages)
+		err = ethChain.SetReceiver(subMessages, nil)
 		if err != nil {
 			return nil, err
 		}
 
-		err = relayChain.SetSender(subMessages, nil, beefyMessages)
+	}
+
+	parachainCommitmentRelayer := &parachaincommitmentrelayer.Worker{}
```

Comment:
nit: My poor eyes. Prefer short variable names such as `relayer`.

Reply:
yeh, now that things are in their own workers, everything is contextual and so we can use short names everywhere

we can do a follow up code quality/code standards refactor PR after all this stuff is merged i think